jetty-server/src/main/java/org/eclipse/jetty/server/handler/ErrorHandler.java

```diff
@@ -429,23 +434,31 @@ private void writeErrorPlain(HttpServletRequest request, PrintWriter writer, int
 
     private void writeErrorJson(HttpServletRequest request, PrintWriter writer, int code, String message)
     {
-        writer
-            .append("{\n")
-            .append("  url: \"").append(request.getRequestURI()).append("\",\n")
-            .append("  status: \"").append(Integer.toString(code)).append("\",\n")
-            .append("  message: ").append(QuotedStringTokenizer.quote(message)).append(",\n");
+        Throwable cause = (Throwable)request.getAttribute(Dispatcher.ERROR_EXCEPTION);
         Object servlet = request.getAttribute(Dispatcher.ERROR_SERVLET_NAME);
+        Map<String,String> json = new HashMap<>();
+
+        json.put("url", request.getRequestURI());
+        json.put("status", Integer.toString(code));
```

Comment:
Just a nitpick, but why converting the code to a String here?

Reply:
The `Map<String,String> json` above would need to change to `Map<String,Object> json` then, right?

Reply:
because the Map is <String,String>